modules/storage-persistent-storage-reclaim-manual.adoc

```diff
@@ -33,7 +33,7 @@ ifdef::openshift-dedicated[]
 The associated storage asset in the external infrastructure, such as an AWS EBS or GCE PD volume, still exists after the PV is deleted.
 endif::openshift-dedicated[]
 ifdef::openshift-rosa[]
-The associated storage asset in the external infrastructure, such as an AWS EBS volume, still exists after the PV is deleted.
+The associated storage asset in the external infrastructure, such as an Amazon EBS volume, still exists after the PV is deleted.
```

Comment:
Revoked.

Reply:
Updated, as per the https://github.com/openshift/openshift-docs/pull/59650#discussion_r1290144425